mypy/build.py

```diff
@@ -2283,7 +2291,8 @@ def find_module_and_diagnose(manager: BuildManager,
                     and not options.follow_imports_for_stubs)  # except when they aren't
                 or id == 'builtins'):  # Builtins is always normal
             follow_imports = 'normal'
-
+        if quick:
+            return (path, follow_imports)
```

Comment:
This is a little odd -- it's really just skipping the next if-elif block (up to line 2309 below). Would it be clearer to write it as
```py
if quick:
    pass
elif follow_imports == 'silent':
    ...
elif ...
```
? Alternatively
```py
if not quick:
    if follow_imports == 'silent':
        ...
    elif ...
```